index.js

```diff
@@ -28,7 +28,7 @@ server({ port: PORT }, [
       })
     }
   }),
-  get('/export-sizes', async ctx => {
+  get('/exports-sizes', async ctx => {
```

Comment:
Not sure what's going on here, this is the path the frontend is using.

Reply:
This is only a dev server — the frontend uses its own routes and server. But makes sense to synchronize here.